src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```
@@ -132,15 +158,19 @@
      * and field provided with -GF. Note that, in moltenized output, all rows will have 'site' as the value of the 'Sample' column.
      */
     @Advanced
-    @Argument(fullName="moltenize", shortName="moltenize", doc="If provided, we will produce molten output", optional=true)
+    @Argument(fullName="moltenize",
+            shortName="moltenize",
+            doc="Produce molten output", optional=true)
     private boolean moltenizeOutput = false;
 
     /**
      * By default, this tool will write out NA values indicating missing data when it encounters a field without a value in a record.
      * If this flag is added to the command, the tool will instead exit with an error if missing data is encountered.
      */
     @Advanced
-    @Argument(fullName="errorIfMissingData", shortName="EMD", doc="If provided, we will require every record to contain every field", optional=true)
+    @Argument(fullName="error-if-missing-data",
+            shortName="EMD",
+            doc="Disallow missing data", optional=true)
```

Maybe "Fail on missing data"?

---

done